File: DiplomAPI/Models/Product.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DiplomAPI.Models
{
    public partial class Product
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Price { get; set; }
        public int QuantityAvailable { get; set; }
    }
}

File: app/src/main/java/com/example/mptfood/Models/Product.kt
package com.example.mptfood.Models
import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Product
    (
        var ID : Int = 0,
        var Name: String? = null,
        var Price: Int = 0,
        var QuantityAvailable: Int = 0

    ) : Parcelable